src/rkopenmdao/runge_kutta_integrator.py

```diff
@@ -239,7 +243,34 @@ def initialize(self):
             types=dict,
             default={},
             desc="""Additional options passed the the checkpointer. Valid options depend
-            on the used checkpointing_type""",
+                   on the used checkpointing_type""",
+        )
+
+        self.options.declare(
+            "error_estimator",
+            types=ErrorEstimator,
+            default=SimpleErrorEstimator,
+            desc="""Error Estimator that is to be used in the ErrorController class. can either be SimpleErrorEstimator
+            or ImprovedErrorEstimator"""
+        )
```

Comment:
You probably also need to define error_estimator_options in that case, since user set options can't be propagated elsewise.

Reply:
Using `isinstance` does not work, since you don't pass an instance of a subclass of ErrorEstimator, but a class itself. For that, there is `issubclass`.